packages/app/src/services/confetti.js

```diff
@@ -0,0 +1,141 @@
+import p5 from 'p5'
+
+/**
+ * A p5.js 'sketch' that renders the confetti pop animation.
+ * Further information about p5: https://p5js.org
+ * @param {*} sketch
```

Comment:
If you provide JSDoc comment with a `@param` tag, please add a parameter description too, e.g. like this:
```
 * @param {*} sketch a short explanation on what the parameter is
```
The syntax is
```
@param {ParameterType} parameterName Description text
```
If no description nor the type are provided, the `@param` tag is useless and should be removed (or more information added).

For further information, see [`@param`](https://jsdoc.app/tags-param.html)